### DISCUSSION TITLE
version of macOS for guest not displayed

### DISCUSSION THREAD

**Discussion by xgrep285** [utmapp/UTM#4758] [UTM Help (macOS)] [answered]
I've installed UTM 3, 4.0.9, and 4.2.1 on macOS Big Sur, Monterey, and Ventura, and on all of them there are no guest architectures displayed when creating a new Virtualization guest, as shown in the attached screen shot. Maybe I'm misunderstanding where these choices can be found?

<img width="453" alt="Screenshot 2022-12-04 at 09 11 45" src="https://user-images.githubusercontent.com/119807812/205505628-b7610309-291f-4b89-aa1c-629bc1d98a17.png">

**Comment by osy** [comment, marked as answer]
You can’t change architecture/CPU when using virtualization. The meaning of virtualization is the use of your current hardware.

**Reply by xgrep285** [reply]
Thank you very much. Of course you're right. I formulated my question incorrectly. What I meant to ask is: How do I specify a macOS version for the guest? When I select Other, the only thing that I can do is provide an iso installer. It's not clear whether this selects settings that are appropriate for that version of macOS.

**Reply by osy** [reply]
https://docs.getutm.app/guest-support/macos/#ipsw

**Reply by xgrep285** [reply]
Thank you, I had read that section, but misunderstood the following sentence:

"macOS guests are supported on Apple Silicon hosts running macOS 12 or higher."

I interpreted this to mean "Apple Silicon hosts require macOS 12 or higher to support macOS guests; Intel silicon hosts can run lower versions of macOS"

I was looking for the macOS selections described by:

In the [new VM wizard](https://docs.getutm.app/basics/basics/), select “Virtualization” and then “macOS 12+” to install macOS as a guest.

Since I have "Virtualize" as a choice, I was expecting to see a selection of macOS versions.

The doc could be easily improved to make clear that Apple Silicon is required, and that macOS is not supported on Intel silicon Macs. I know that the sentence as written can be understood to mean that, but it can also be understood differently.

Thank you again for clarifying the New Guest behavior.
